packages/profile-sync-controller/src/sdk/authentication-jwt-bearer/flow-srp.ts

```diff
@@ -240,6 +254,35 @@ export class SRPJwtBearerAuth implements IBaseAuth {
     }
   }
 
+  async #loginWithRetry(entropySourceId?: string): Promise<LoginResponse> {
+    // Allow max 2 attempts: initial + one retry on 429
+    for (let attempt = 0; attempt < 2; attempt += 1) {
```

Comment:
The max attempt should be a constant IMO or maybe coming from the `options` like `cooldownDefaultMs`?